app/commands/source/add_category_command.rb

```diff
@@ -29,37 +29,42 @@ def execute
         validate_id!
         create_category!
         update_data_files!
-        sync_localizations!
       end
     end
 
     private
 
     def find_parent!
-      @parent = Category.find_by(id: params[:parent])
+      @parent = Category.find_by(id: params[:parent_id])
       return @parent if @parent
 
-      logger.fatal("Parent category `#{params[:parent]}` not found")
+      logger.fatal("Parent category `#{params[:parent_id]}` not found")
       exit(1)
     end
 
     def validate_id!
       params[:id] ||= @parent.next_child_id
-      return if params[:id].start_with?(params[:parent])
+      return if params[:id].start_with?(params[:parent_id])
 
-      logger.fatal("ID `#{params[:id]}` does not start with parent ID `#{params[:parent]}`")
+      logger.fatal("ID `#{params[:id]}` does not start with parent ID `#{params[:parent_id]}`")
```

Comment:
oh right on. I vote to reduce any redundancy... otherwise I (or someone else) may be tempted to add something similar in the future here forgetting it is captured on the category. 😅